package-lock.json

```diff
@@ -4393,6 +4393,7 @@
 			"version": "8.48.0",
 			"resolved": "https://registry.npmjs.org/eslint/-/eslint-8.48.0.tgz",
 			"integrity": "sha512-sb6DLeIuRXxeM1YljSe1KEx9/YYeZFQWcV8Rq9HfigmdDEugjLEVEa1ozDjL6YDjBpQHPJxJzze+alxi4T3OLg==",
+			"deprecated": "This version is no longer supported. Please see https://eslint.org/version-support for other options.",
```

Comment:
While this is not an important change, it does make OED redo the npm install. I'm not sure how it was changed but could you please remove it.

Reply:
Unfortunately, the manual changes appear to have changed the date on the file so the OED install thinks it is updated even though the change is gone. I tried to fix but could not do it cleanly. Thus, I'm letting it go and people may see an npm install.